CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### Changed
 - bump starknet-rs dependency
 - replaced starknet-devnet with starknet-devnet-rs for tests
+- align scarb version in tests with forge
```

Comment:
Does it make sense to include it in changelong if it's only related to tests?

Reply:
Following discussin in https://github.com/foundry-rs/starknet-foundry/pull/812, removed this :+1: